spack-environments/archer2/compute-node/spack.yaml

```diff
@@ -283,6 +323,12 @@ spack:
       externals:
       - spec: pkg-config@0.29.2
         prefix: /usr
+    python:
+      externals:
+      - spec: python@2.7.18+bz2+ctypes~dbm+nis+pyexpat+pythoncmd+readline+sqlite3+ssl~tix~tkinter+uuid+zlib
+        prefix: /usr
+      - spec: python@3.6.12+bz2+ctypes~dbm+lzma+nis+pyexpat~pythoncmd+readline+sqlite3+ssl~tix~tkinter+uuid+zlib
+        prefix: /usr
```

Comment:
You may want to look at the Python 3 environment in the `cray-python` module for a more complete installation